db/migrate/20200604122917_create_orders.rb

```diff
@@ -0,0 +1,13 @@
+class CreateOrders < ActiveRecord::Migration[6.0]
+  def change
+    create_table :orders do |t|
+      t.references :user, null: false, foreign_key: true
+      t.references :payment, null: false, foreign_key: true
```

Comment:
not sure if the payment references should be `null:false` else what will happen in the case that the order is not transacted yet?
saw there is column: completed, so if completed == true, then references :payment ya?

read up on this: https://stackoverflow.com/questions/32305821/rails4-migration-adding-null-false-to-an-add-reference-method
maybe just need to remove `null: false`

Reply:
Notes from Prima:

- Line 5 change to `null: true`, cannot just remove `null: false` because `null: true` is the default if not specified.

- In the model, add `optional: true` for the reference field

Reply:
Notes on how to change field/column in a new migration [StackOverflow](https://stackoverflow.com/questions/5966840/how-to-change-a-nullable-column-to-not-nullable-in-a-rails-migration/5967073#5967073)

Reply:
where to add optional: true?